services/ingest/tests/test_github_app_auth.py

```
@@ -0,0 +1,113 @@
+import time
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import httpx
+import jwt
+import pytest
+
+from app import github_app_auth
+
+
+@pytest.fixture(autouse=True)
+def _clear_token_cache():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_clear_token_cache`

(ANN202)